docs/devel/getting-builds.md

```diff
@@ -0,0 +1,24 @@
+# Getting Kubernetes Builds
+
+You can use [get-build.sh](../../hack/get-build.sh) to, or use as a reference on how to get the most recent builds with curl. With `get-build.sh` you can grab the most recent stable build, the most recent release candidate, or the most recent build to pass our ci and gce e2e tests (essentially a nightly build).
```

Comment:
minor stylistic nits: the comma after "to" is awkward. and I'm not sure whether GCE should be capitalized or not.